features/waarden.feature

```diff
@@ -0,0 +1,160 @@
+# language: nl
+
+Feature: leveren van waarden bij een WOZ-object
+  Per waardepeildatum wordt één waarde opgenomen.
+  Voor een waardepeildatum wordt de waarde genomen met de meest recente ingangsdatum met betrekking tot die waardepeildatum.
```

Comment:
Ik concludeer hieruit:

- er kunnen waarden voorkomen met dezelfde waardepeildatum en verschillende ingangsdatum. In dit geval geldt voor de betreffende waardepeildatum de waarde met de meest recente ingangsdatum
- er kunnen waarden met dezelfde waardepeildatum en dezelfde ingangsdatum. In dit geval geldt voor de betreffende waardepeildatum de waarde met de meest recente beschikkingsdatum

Klopt dit?

Reply:
ja, dat klopt

Reply:
ik heb dit nog iets verduidelijkt in de titels van de scenario's